.github/workflows/update-vulnerabilties.yaml

```diff
@@ -0,0 +1,71 @@
+name: Update Vulnerabilities
+on:
+  push:
+    branches:
+      - "yli3/**"
+  schedule:
+    - cron: "0 */3 * * *"
+
+jobs:
+  read-versions:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.output-matrix.outputs.matrix }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Generate matrix JSON
+        id: output-matrix
+        run: |
+          EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
+          echo "matrix<<$EOF" >> $GITHUB_OUTPUT
+          sed -n '/^\s*\(#.*\)\?$/!p' scanner/updater/version/RELEASE_VERSION | jq -Rs '{ versions: ( sub("\n$"; "")|split("\n") ) }' | tee -a "$GITHUB_OUTPUT"
```

Comment:
I tried this locally and got the following:

```
{
  "versions": [
    "# This file lists the ACS versions supported by Scanner V4 and Scanner V4 Updater.",
    "# Since Scanner V4 shares the same version as ACS,",
    "# Scanner V4 Updater updates vulnerability data for all versions of Scanner V4 listed in this file",
    "# and uploads that data to Google Storage.",
    "# Various versions of Scanner V4 matchers will import vulnerability data based on their respective versions from Google Storage.",
    "4.0.0",
    "3.74.4",
    "4.1.x-476-g1aa6557244",
    "4.1.x-786-g1a0528a30c",
    "4.1.x-787-g9025f208b7"
  ]
}
```

I don't think the comments are meant to be there, right? If not, I'm wondering if we even need the comments (I suggested removing them and adding a README in a separate review comment)?

Reply:
Probably your `sed` is non-gnu. It should be fine because this workflow runs in `ubuntu-latest`.

Reply:
The comments should be there and should be ignored.